lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -301,17 +301,11 @@ html.video-fullscreen {
           &.fa-arrow-circle-right {
             top: 30%;
           }
-        }
-        &:last-child {
-          > .fa {
+          &.exam-without-proctoring {
```

Comment:
The way this works is that the edx-proctoring repo renders a button with this class, and then this Sass rule in edx-platform changes the way it renders to not show as a blue button. The old code always showed the last button as gray, but that doesn't work when we have only a single button.

I think I'll rename it as ``secondary-action`` as the only meaning is that this is not the primary button.